pallets/runtime/tests/src/settlement_test.rs

```diff
@@ -1558,18 +1562,22 @@ fn failed_execution() {
             Settlement::instruction_leg_status(instruction_counter, 1),
             LegStatus::PendingTokenLock
         );
+
+        // Check that tokens are locked for settlement execution.
         assert_eq!(
             Portfolio::locked_assets(PortfolioId::default_portfolio(alice_did), &ticker),
             amount
         );
 
+        // Ensure balances have not changed.
         assert_eq!(Asset::balance_of(&ticker, alice_did), alice_init_balance);
         assert_eq!(Asset::balance_of(&ticker, bob_did), bob_init_balance);
         assert_eq!(Asset::balance_of(&ticker2, alice_did), alice_init_balance2);
         assert_eq!(Asset::balance_of(&ticker2, bob_did), bob_init_balance2);
```

Comment:
Also seems like copypasta... let's call the closure `ensure_balances_unchanged();`